app/views/admin/service_announcements/_menu.html.haml

```diff
@@ -1,21 +1,20 @@
 .ui.fluid.vertical.menu.sticky{ data: { behavior: "sticky", context: "#stickContext", offset: 75 } }
   .item
-    .header
-      Service Announcements
+    .header Service Announcements
+
+  .item
+    .ui.transparent.icon.input
+      = form_tag admin_service_announcements_path, method: :get do
+        = search_field_tag "q", params[:q], class: "ui prompt", placeholder: "Search ...", aria: { label: "search" }
+      %i.search.link.icon
 
   = active_link_to admin_service_announcements_path, active: :exclusive, class: "item" do
     Index
-    - if count
+    - if defined?(count) && count
```

Comment:
this is weird

Reply:
Yeah :< But needed, if I just have it as the old `if count` then it fails with undefined method count if I don't pass it in through `locales: {}`